feature/aft/aft_base/otg_tests/aft_prefixcounters/README.md

```diff
@@ -0,0 +1,178 @@
+# AFT-2.1: AFTs Prefix Counters
+
+## Summary
+
+IPv4/IPv6 prefix counters
+
+## Testbed
+
+* atedut_2.testbed
+
+## Test Setup
+
+### Generate DUT and ATE Configuration
+
+Configure DUT:port1 for IS-IS session with ATE:port1
+*   IS-IS must be level 2 only with wide metric.
+*   IS-IS must be point to point.
+*   Send 1000 ipv4 and 1000 ipv6 IS-IS prefixes from ATE:port1 to DUT:port1.
+
+Establish eBGP sessions between ATE:port1 and DUT:port1.
+*   Configure eBGP over the interface ip.
+*   Advertise 1000 ipv4,ipv6 prefixes from ATE port1 observe received prefixes at DUT.
+
+### Procedure
+
+*   Gnmi set with REPLACE option to push the configuration DUT.
+*   ATE configuration must be pushed.
+
+### verifications
+
+*   BGP routes advertised from ATE:port1 must have 1 nexthop.
+*   IS-IS routes advertised from ATE:port1 must have one next hop.
+*   Use gnmi Subscribe with ON_CHANGE option to /network-instances/network-instance/afts.
+*   Verify afts prefix entries using the following paths with in a timeout of 30s.
+
+/network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/prefix,
+/network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/prefix
```

Comment:
Why are these paths here? Should these two lines be removed?

Reply:
that is needed for validation.